### PR TITLE
Fix tests and package update

### PR DESCRIPTION
**Pull request by flightlevel**
Fix tests and package update

### CODE REVIEW

**Comment by flightlevel** [comment]
Weird how the tests work every time on my PC, but fail on both Appveyor and Travis ???

```
PS D:\IrcDotNet\test\IrcDotNet.Test> dotnet build
Microsoft (R) Build Engine version 15.6.82.30579 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restoring packages for D:\IrcDotNet\source\IrcDotNet\IrcDotNet.csproj...
  Restoring packages for D:\IrcDotNet\test\IrcDotNet.Test\IrcDotNet.Test.csproj...
  Generating MSBuild file D:\IrcDotNet\source\IrcDotNet\obj\IrcDotNet.csproj.nuget.g.props.
  Generating MSBuild file D:\IrcDotNet\source\IrcDotNet\obj\IrcDotNet.csproj.nuget.g.targets.
  Restore completed in 857.09 ms for D:\IrcDotNet\source\IrcDotNet\IrcDotNet.csproj.
  Generating MSBuild file D:\IrcDotNet\test\IrcDotNet.Test\obj\IrcDotNet.Test.csproj.nuget.g.props.
  Generating MSBuild file D:\IrcDotNet\test\IrcDotNet.Test\obj\IrcDotNet.Test.csproj.nuget.g.targets.
  Restore completed in 993.18 ms for D:\IrcDotNet\test\IrcDotNet.Test\IrcDotNet.Test.csproj.
  IrcDotNet -> D:\IrcDotNet\source\IrcDotNet\bin\Debug\netstandard1.5\IrcDotNet.dll
  IrcDotNet.Test -> D:\IrcDotNet\test\IrcDotNet.Test\bin\Debug\netcoreapp1.1\IrcDotNet.Test.dll

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:05.35




PS D:\IrcDotNet\test\IrcDotNet.Test> dotnet run
NUnitLite 3.10.1 (.NET Standard 1.6)
Copyright (c) 2018 Charlie Poole, Rob Prouse

Runtime Environment
   OS Version: Microsoft Windows 10.0.16299
  CLR Version: .NET Core 4.6.25211.01

Test Files
    D:/IrcDotNet/test/IrcDotNet.Test/bin/Debug/netcoreapp1.1/IrcDotNet.Test.dll

Run Settings
    Work Directory: D:\IrcDotNet\test\IrcDotNet.Test
    Internal Trace: Off

Test Run Summary
  Overall result: Passed
  Test Count: 1, Passed: 1, Failed: 0, Warnings: 0, Inconclusive: 0, Skipped: 0
  Start time: 2018-04-08 06:30:55Z
    End time: 2018-04-08 06:32:10Z
    Duration: 74.573 seconds

Results (nunit3) saved as D:\IrcDotNet\test\IrcDotNet.Test\TestResult.xml
```

**Comment by chrboe** [comment]
I think @paralin once mentioned that we should move to a single CI system anyways, so I'm guessing that this should be just fine as long as it works. 

Input, @paralin?

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: csharp
 dist: trusty
 mono: none
-dotnet: 1.0.1
+dotnet: 1.0.4
```
Comment (chrboe) [inline .travis.yml]: Is 1.0.4 the highest version we can move to? As I see it, we should bump the version to the highest possible supported one.

**Comment by paralin** [comment]
We can get rid of appveyor imo.

**Comment by paralin** [comment]
Okay read through this. I realize appveyor has windows support. I'm ok with keeping it and killing Travis in the short term, but we need to target the new dotnet core as well and make sure we are targeting Linux. We can do this in multiple PRs if needed.

**Comment by flightlevel** [comment]
Happy medium
-Continue to use both Appveyor and Travis-CI
-Appveyor which is using the .NET core runtime does build and tests
-Travis-CI runs just a build (no tests) so you can be sure it builds on linux

File: .travis.yml (hunk 1 above)
Comment (flightlevel) [reply .travis.yml]: updated to latest .NET core SDK

**Comment by paralin** [comment]
Thanks for the work. Will review

**Comment by junglebus** [comment]
Keen to see this get merged, so that other PRs can be accepted

**Comment by flightlevel** [comment]
any update?

**Comment by paralin** [comment]
Apologies for letting this get away from me, I'll be sure to get this going again Monday.

**Pull request merged**